[Docs] Add that default conda env in CE is not persistent (#8288)
mlrun/mlrun: docs/install/kubernetes.md

## Configuring the user Jupyter conda environment

> The default Jupyter comes with a conda env named `mlrun`. This conda is installed on a non-persistent path.

Reviewer: ```suggestion
The default Jupyter comes with a conda env named `mlrun`. This conda is not persistent.
```

> The default Jupyter comes with aß conda env named `mlrun`. This conda is installed on a non-persistent path `/opt/conda/envs/mlrun`.

Reviewer: ```suggestion
The default Jupyter comes with a conda env named `mlrun`. This conda is installed on a non-persistent path.
```
I dont think we need to mentioned to the user the specific path just that the conda is not persistent

> If you install Python on this conda env, and then the Jupyter pod gets restarted or deleted, those packages will be deleted.

Reviewer: ```suggestion
If you install any packages on this conda env, and then the Jupyter pod gets restarted or deleted, those packages will be deleted.
```

> If you install Python packages on this conda env, and then the Jupyter pod gets deleted or moved to another node, those Python packages will be deleted.

Reviewer: ```suggestion
If you install Python on this conda env, and then the Jupyter pod gets restarted or deleted, those packages will be deleted.
```
The user may have installed other packages, and not just Python, and I think a restart or deletion is clearer than placing the pod in a diffrent node
